airflow/models/taskinstance.py

```diff
@@ -154,6 +154,7 @@ class TaskInstance(Base, LoggingMixin):
     unixname = Column(String(1000))
     job_id = Column(Integer)
     pool = Column(String(50), nullable=False)
+    pool_capacity = Column('pool_capacity', Integer, default=1)
```

Comment:
Since the attribute name is the same as the `Column.name`, I suggest removing the `Column.name` and use the default mapping to be consistent with the rest.

Reply:
I have modified the code as suggested.